uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -29,6 +29,7 @@ define([
       var mode = ''; // can be edit, create, view
       var role = '';
       var initialCommentsLength = 0; // number of comments when the request model is loaded
+      var NO_ENTITLEMENT_ERROR = 'No entitlement';
```

Comment:
As this variable stores a constant string value, its made in all caps. which is a common coding practise.